crates/cargo-util-schemas/src/schema.rs

```diff
@@ -0,0 +1,56 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::string::String;
+use toml::Value as TomlValue;
+#[derive(Serialize, Deserialize)]
+pub struct TomlValueWrapper(pub TomlValue);
+#[test]
+fn dump_schema() {
+    let schema = schemars::schema_for!(crate::manifest::TomlManifest);
+    let dump = serde_json::to_string_pretty(&schema).unwrap();
+    snapbox::assert_data_eq!(dump, snapbox::file!("../schema.json"));
```

Comment:
We're going to have more than one schema file.  We should qualify which schema this is